frontend/lib/src/components/core/Block/Block.tsx

```diff
@@ -219,7 +219,7 @@ const VerticalBlock = (props: BlockPropsWithoutWidth): ReactElement => {
       new ResizeObserver(([entry]) => {
         // We need to determine the available width here to be able to set
         // an explicit width for the `StyledVerticalBlock`.
-        setWidth(entry.target.getBoundingClientRect().width)
+        setWidth(Math.round(entry.target.getBoundingClientRect().width))
```

Comment:
not 100% sure if `round` is better than `floor` here. `floor` feels like a slightly safer option since with that the calculated width is guaranteed to always stay at or below parent width. `round` might have some strange side effect depending on the browser.